jsonnet/kube-prometheus/alertmanager/alertmanager.libsonnet

```diff
@@ -40,6 +40,10 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
         ],
       },
       replicas: 3,
+      resources: {
+        imits: { "cpu": "100m",  "memory": "100Mi" },
```

Comment:
Small typo here, it should be `limits`.

Also, to be consistent with the rest of the file, the `limits` and `requests` objects should be formatted as

```jsonnet
{ cpu: '100m', memory: '100Mi' }
```

Reply:
thank you for your review, I have fixed it ;)